test/functionalTest/cases/1308_subscription_cache/cache_reset_from_db.test

```diff
@@ -0,0 +1,667 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Possible Bug in Subscription Cache when reloading from database
+
+--SHELL-INIT--
+dbInit CB
+dbInit ftest-t1
```

Comment:
However, you are using a name that depends of a env var CB_DATABASE_NAME (ftest), so if somebody changes CB_DATABASE_NAME this instruction breaks.

Maybe the good solution would be to have a function to initialize tenants. Something like:

dbInitTenant CB f1

However, not for this PR. I will open an issue about it to address it in the future.